src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -66,5 +66,5 @@ void sort_nullInput() {
         PostSorter authorPostSorter = new AuthorPostSorter();
         List<UserPost> nullInput = Arrays.asList();
         Assertions.assertEquals(nullInput, authorPostSorter.sort(null, SortOrder.ASC));
-    }
+    }*/
```

Comment:
Bad practice to comment out test.
Use annotation (at)Ignore or (at)Disabled and add a comment to why this is disabled